src/main/java/com/ldtteam/structurize/network/messages/NotifyClientAboutStructurePacks.java

```diff
@@ -0,0 +1,77 @@
+package com.ldtteam.structurize.network.messages;
+
+import com.ldtteam.structurize.storage.ClientStructurePackLoader;
+import com.ldtteam.structurize.storage.StructurePackMeta;
+import net.minecraft.network.FriendlyByteBuf;
+import net.minecraftforge.fml.LogicalSide;
+import net.minecraftforge.network.NetworkEvent;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Notify the client about the structure packs on the server side.
+ */
+public class NotifyClientAboutStructurePacks implements IMessage
```

Comment:
ClientStructurePacksMessage ?

Reply:
No, it notifies the client, so would more be of a ServerStructurePacksMessage

Reply:
I'd see client in name as a message dedicated for the client to receive^^